presto-main/src/main/java/com/facebook/presto/operator/OperatorContext.java

```diff
@@ -510,14 +499,6 @@ public OperatorStats getOperatorStats()
         return visitor.visitOperatorContext(this, context);
     }
 
-    private long currentThreadUserTime()
-    {
-        if (!collectTimings) {
-            return 0;
-        }
-        return THREAD_MX_BEAN.getCurrentThreadUserTime();
```

Comment:
What are the situations where user time was more useful than CPU time?

If we are still concerned about losing potentially useful information, as an alternative , we could leave this, but behind different toggle, `collectUserTiming` or something.

(i don't see this necessity, just asking)

Reply:
It seems like this was added just because the method was there. We don't use this metric internally.